static/app/components/events/interfaces/breadcrumbs/breadcrumb/data/linkedEvent.tsx

```diff
@@ -67,10 +67,14 @@ function LinkedEvent({orgSlug, eventId, route, router}: Props) {
         return;
       }
 
-      addErrorMessage(
-        t('An error occurred while fetching the data of the breadcrumb event link')
+      const errorMessage = t(
+        'The endpoint %s is being used very quickly. The limit is 1/1s',
```

Comment:
Including the endpoint will make the message very noisy, it's not needed. Also because we're using it as an error message, will it still be grouped properly? The message will be different everytime due to the dynamic endpoint.

Reply:
Also I don't think the assumption here is correct in that every error is due to the API request rate, it could fail for other reasons.

Reply:
sure, but it looks like the backend is not sending the correct error message so I thought we could use this message for now until we work on the endpoint again

Reply:
I've updated the code to use `handleXhrErrorResponse`